### PR TITLE
fix layout and remove animations from privacy policy page

### LINKED ISSUE #3344: [Frontend V2] Modifications for Privacy policy page
### Description

Screen Shot 2021-03-24 at 12 28 41 PM

Following are the modifications for Privacy policy page:

- [ ] Center the page content for large screens
- [ ] Start the sidebar from the top of the page, currently the sidebar is centered on the page it should start from the top
- [ ] Remove the rocket, cloud and birds animation from the page

### PR DESCRIPTION
**Pull request by burnerlee**
resolves #3344

### CODE REVIEW

**Comment by burnerlee** [comment]
Screenshot_2021-03-25_13-43-11
Screenshot_2021-03-25_13-43-27
Screenshot_2021-03-25_13-43-38
Screenshot_2021-03-25_13-43-48

**Comment by Ram81** [comment]
@burnerlee can you share screenshot of the page on larger screen

**Comment by burnerlee** [comment]
Screenshot_2021-03-28_17-22-55
yeah sure, sorry i missed to attach this one @Ram81

File: frontend_v2/src/app/components/privacy-policy/privacy-policy.component.scss [hunk 1]
```diff
@@ -93,3 +93,18 @@
...
+}
+
+@include screen-large {
+  .privacy-container {
+    padding-left:0;
+    padding-right: 0;
+    margin-right: auto;
+    margin-left: auto;
+  }
+}
```
Comment (Ram81) [inline frontend_v2/src/app/components/privacy-policy/privacy-policy.component.scss:110]: add newline

**Review by Ram81 — CHANGES_REQUESTED** [review]
@burnerlee minor change, rest of it LGTM

**Comment by burnerlee** [comment]
@Ram81 made the change

**Review by Ram81 — APPROVED** [review]
LGTM 👍

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**